testnet/stacks-node/src/globals.rs

```diff
@@ -355,6 +359,28 @@ impl<T> Globals<T> {
         }
     }
 
+    /// Get the last burnchain config
+    pub fn get_last_burnchain_config(&self) -> Option<BurnchainConfig> {
+        match self.last_burnchain_config.lock() {
+            Ok(last_burnchain_config) => (*last_burnchain_config).clone(),
+            Err(_e) => {
+                error!("FATAL; failed to lock last burnchain config");
+                panic!();
+            }
+        }
+    }
+
+    /// Set the last burnchain config
+    pub fn set_last_burnchain_config(&self, burnchain_config: BurnchainConfig) {
+        match self.last_burnchain_config.lock() {
+            Ok(ref mut last_burnchain_config) => **last_burnchain_config = Some(burnchain_config),
+            Err(_e) => {
+                error!("FATAL; failed to lock last burnchain config");
+                panic!();
+            }
+        }
+    }
```

Comment:
It doesn't seem like this function is used anywhere in this PR. Is that intentional?

Reply:
Ah, yeah I definitely did mean to, and I've updated this PR to use it. Without using this, the "did burnchain config change" check would always have returned true (after updating it once).